src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -289,6 +292,13 @@ private static string GetUri(HttpRequest request)
             return builder.ToString();
         }
 
+        private static void SetStatusFromHttpStatusCode(Activity activity, int statusCode)
```

Comment:
Might be worth inlining `SetStatusFromHttpStatusCode` & `TryGetGrpcMethod` they are only called a couple of places.

Reply:
I also applied inline attribute to `AddGrpcAttributes`. Still kind of trying to feel out when best to apply the inlining attribute. I guess it's technically a balance between code size and performance. Does your intuition usually guide you to apply it to methods with few callers?

Reply:
> I guess it's technically a balance between code size and performance.

Totally! I think for OpenTelemetry our mission is to have as little overhead on the hosting process as possible so I tend to lean towards performance over binary size. I kind of ask myself: Is this hot path? Is this method really just to keep the code clean? Is it called in only a couple of spots (or is it really tiny)? If the answer is yes to all of that, I usually add the inline hint.